be/src/exec/pipeline/exchange/sink_buffer.cpp

```diff
@@ -355,7 +356,7 @@ Status SinkBuffer::_try_to_send_rpc(const TUniqueId& instance_id, const std::fun
             } else {
                 _try_to_send_rpc(ctx.instance_id, [&]() {
                     _process_send_window(ctx.instance_id, ctx.sequence);
-                    _update_network_time(ctx.instance_id, ctx.send_timestamp, result.receive_timestamp());
+                    _update_network_time(ctx.instance_id, ctx.send_timestamp, result.receive_process_time());
```

Comment:
If the user is upgrading in grayscale, the result received from the old version of BE does not have this field, should we deal with the compatibility issue?

Reply:
I think we don't have to, you always get a wrong profile in this situation no matter how you do.

Reply:
We have to always keep these two calcuation methods which is a high cost I think. So maybe we can choose to not keep compatible.

Reply:
> We have to always keep these two calcuation methods if we take compatibility into consideration which is a pretty high cost I think. So maybe we can choose to not keep compatible, which results in a wrong profile.

I think it's not a big problem and can be acceptable, but we need to tell users in release note that the exchange profile may be abnormal during the upgrade…

Reply:
would it may cause crash?